oasislmf/manager.py

```diff
@@ -420,6 +421,16 @@ def generate_oasis_files(
             exposure_fp, _keys_fp, exposure_profile=exposure_profile
         )
 
+        # Write exposure summary file
```

Comment:
A change is required on line 395 in the manager to support the deterministic loss generation workflow - in that scenario there is no model and the keys are auto-generated, and there are no keys errors by definition. So an empty keys errors file must be written out anyway.

I will test this, and if it works, update the PR directly.